### PR TITLE
Add supportivekoala API

### PR DESCRIPTION
**Pull request by nadalfederer**
- [x] My submission is formatted according to the guidelines in the [contributing guide](/CONTRIBUTING.md)
- [x] My addition is ordered alphabetically
- [x] My submission has a useful description
- [x] The description does not end with punctuation
- [x] Each table column is padded with one space on either side
- [x] I have searched the repository for any relevant issues or pull requests
- [x] Any category I am creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -424,6 +424,7 @@ API | Description | Auth | HTTPS | CORS |
 | [SHOUTCLOUD](http://shoutcloud.io/) | ALL-CAPS AS A SERVICE | No | No | Unknown |
 | [StackExchange](https://api.stackexchange.com/) | Q&A forum for developers | `OAuth` | Yes | Unknown |
 | [Statically](https://statically.io/) | A free CDN for developers | No | Yes | Yes |
+| [Supportivekoala](https://developers.supportivekoala.com/) | Autogenerate images with template | apiKey | Yes | Yes |
```
Comment (matheusfelipeog) [inline README.md:427]: auth value is not enclosed with \`backticks\`

```suggestion
| [Supportivekoala](https://developers.supportivekoala.com/) | Autogenerate images with template | `apiKey` | Yes | Yes |
```

**Review by matheusfelipeog — CHANGES_REQUESTED** [review]
Thanks for your contribution @nadalfederer :)

Can you resolve this review, please?

**Comment by nadalfederer** [comment]
Hi, I fixed this issue

**Review by matheusfelipeog — APPROVED** [review]
Now it's Ok ✔

Thanks for your update @nadalfederer ;)

**Pull request merged**